Flexibus/#500/edit profile ui changes

jkbe opened the pull request:
Closes #500.
- buttons to save changes on edit pages now show "Speichern" instead of "Aktualisiere \<model\>" everywhere
- label and text-input sizes and alignment should be fixed for all forms, not just the one on the edit user page

Commit:
Remove telegram logo from form #500

Commit:
Increase form input field text size and align with input label #500

Commit:
Update buttons on edit user profile page #500

Commit:
Merge branch 'dev' into flexibus/#500/edit-profile-UI-changes

jonashering commented on config/locales/de.helpers.yml:
```diff
@@ -40,7 +40,7 @@ de:
       create: "Erstelle %{model}"
       submit: "Speichere %{model}"
-      update: "Aktualisiere %{model}"
+      update: "Speichern"
```
Creates inconsistency between `en.helpers.yml` and `de.helpers.yml`; One requires a model to be set, and one doesn't. Although current focus is on the German version, I guess it's still a good idea to keep existing files clean and up to date. 

Why don't you use the already existent text from `t'(helpers.links.save)` or change the line in all concerned locale files.

jkbe replied:
Yup, I'll adapt the `de.helpers.yml` accordingly.
I decided against using `helpers.links.save` because then I'd have to pass this string manually on all edit pages for all models vs just changing the string which is automatically retrieved in Rail's convention-over-configuration manner when it sees an `<%= f.submit ... %>` on an edit page.

jonashering replied:
Okay, I agree.

jonashering reviewed (changes requested):
The redesign looks neat! Great work, Jakob! 💯 
Just have a small comment...

Commit:
Adapt locale string to match format in other locale #500

Also remove obsolete code because i18n doesn't expect a model name anymore for this string.

jkbe commented:
Please recheck @jonashering

Commit:
Merge branch 'dev' into flexibus/#500/edit-profile-UI-changes
